game/scripts/npc/abilities/ember_spirit_flame_guard.txt

```diff
@@ -56,7 +56,7 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_per_second"                               "30 40 50 60 110 220"
+        "damage_per_second"                               "30 40 50 60 200 350"
```

Comment:
you are buffing both magic and physical builds for ember, personally I'd choose just one. This is a fine change

Reply:
Physical Ember Died with Devastator and Desolater no longer stacking.